consul/consul/spec.yml

```diff
@@ -2,7 +2,7 @@
 name: consul
 version: 0.6.4
```

Comment:
the output of `consul version` returns values set by `git describe --tags --always` as part of the build process. this discrepancy doesn't affect functionality so we'll just have to live with it for now. otherwise, lgtm.